### ISSUE TITLE
Pushing to remote failed

### ISSUE THREAD

**Issue by phearme** [zaggino/brackets-git#1016]
[using Gitlab, when I push to remote repo, the commit is pushed Ok, and I can see my commit on the server, but the "Pushing" window in brackets remains on the screen and doesn't end.]

--------------------------------

*Brackets 1.3.0-16022 (linux), Git 1.9.1 & Brackets-Git 0.14.24*

**Pushing to remote failed**

```
Error: cmd-spawn-timeout: git push --porcelain --progress origin master
```

```
call: git push --porcelain --progress origin master
Error: cmd-spawn-timeout: git push --porcelain --progress origin master
    at timeoutPromise (file:///home/rcohen/.config/Brackets/extensions/user/zaggino.brackets-git/src/Cli.js:219:27)
    at file:///home/rcohen/.config/Brackets/extensions/user/zaggino.brackets-git/src/Cli.js:262:33
```

**Comment by zaggino** [comment]
hi @phearme , try updating your Git to solve this
```
sudo add-apt-repository ppa:git-core/ppa
sudo apt-get update
sudo apt-get install git
```

**Comment by phearme** [comment]
solved updating git `git version 2.3.7`
